src/main/webapp/js/instructorCourseDetails.js

```diff
@@ -49,10 +49,8 @@ function submitFormAjax() {
  * Functions to trigger registration key sending to a specific student in the
  * course.
  * Currently no confirmation dialog is shown.
- * @param courseID
- * @param email
  */
-function toggleSendRegistrationKey(courseID, email) {
+function toggleSendRegistrationKey() {
```

Comment:
The issue for this PR mentioned, it's only being called without passing any parameters in.